src/include/compat/nodes/relation.h

```diff
@@ -0,0 +1,5 @@
+#if PG_VERSION_NUM >= 120000
```

Comment:
Do we really need these `compat/...` header files. Can't we do as we used to, have these ifdefs in the code?

I can see that the number of such files could increase, and the cost of maintaining would increase. So, I'd go for just inlining these changes in the places where they are used.